src/DataGrid.tsx

```diff
@@ -629,11 +629,10 @@ function DataGrid<R, SR, K extends Key>(
   }
 
   function handleCellInput(event: React.KeyboardEvent<HTMLDivElement>) {
-    if (!isCellWithinBounds(selectedPosition)) return;
+    if (!isCellWithinBounds(selectedPosition) || selectedPosition.idx === -1) return;
```

Comment:
If a regular row is selected in a group and we enter a key which is not explicitly handled (like 'a') then we get the following error

![image](https://user-images.githubusercontent.com/1452717/128218348-5ffc16c2-d4ee-47ea-9089-4e5edbe23bbf.png)

Reply:
I think we should revisit `isCellWithinBounds` and it should probably return false if `idx === -1`